src/contexts/bridge.context.tsx

```diff
@@ -682,11 +700,17 @@ const BridgeProvider: FC<PropsWithChildren> = (props) => {
                 overrides
               )
               .then((gasLimit) => {
-                const gasIncrease = gasLimit
+                const gasLimitIncrease = gasLimit
                   .div(BigNumber.from(100))
                   .mul(BRIDGE_CALL_GAS_LIMIT_INCREASE_PERCENTAGE);
 
-                return gasLimit.add(gasIncrease);
+                const permitGasLimitIncrease = BigNumber.from(
+                  BRIDGE_CALL_PERMIT_GAS_LIMIT_INCREASE
+                );
+
+                return tokenSpendPermission.type === "permit"
+                  ? gasLimit.add(gasLimitIncrease).add(permitGasLimitIncrease)
```

Comment:
`add` supports a `BigNumberish` as a param, so we can simplify this a little bit:

```ts
const gasLimitIncrease = gasLimit
  .div(BigNumber.from(100))
  .mul(BRIDGE_CALL_GAS_LIMIT_INCREASE_PERCENTAGE);
const increasedGasLimit = gasLimit.add(gasLimitIncrease)

return tokenSpendPermission.type === "permit"
  ? increasedGasLimit.add(BRIDGE_CALL_PERMIT_GAS_LIMIT_INCREASE)
  : increasedGasLimit
```

Reply:
Yep, much better. Done.